recipe/meta.yaml

```diff
@@ -10,16 +10,16 @@ source:
   sha256: a14c1c5143081701d4fe62283e09cc5ad5336dff3a152de64eb77ee3180e9792
 
 build:
-  number: 0
+  number: 1
   noarch: python
 
 requirements:
   host:
     - python >=3.8
   run:
     - python >=3.8
-    - dask-core {{ version }}
-    - distributed {{ version }}
+    - dask-core =={{ version }}
+    - distributed =={{ version }}
```

Comment:
So one downside of this particular pinning is that we are not able to pick up a hot-fix from `dask-core` or `distributed` (for example `2022.8.1.1`) without a corresponding release.

Reply:
Yeah I was thinking about this - I wasn't sure how impactful this would be to the release process, considering that in the past, hotfixes have generally been done through a new patch release of all 3 packages. However, this obviously would limit our options moving forward.

Do you know of any way we can achieve the intended solution (excluding pre-alpha packages with the same version prefix) without using as tight of a pinning?

Reply:
TIL `dask-core {{ version }}` and `dask-core =={{ version }}` are different -- thanks @charlesbluca! 

Re: hot-fixes, I agree with @charlesbluca that I don't think this will be an issue in practice given our current release process. We've already just issued a new release of all three packages which would, I think, work with the existing changes here. The only other thing that comes to mind is security backports (Xref https://github.com/dask/community/issues/244) which we've yet to ever act on. IIUC this _could_ throw a wrench into the current proposal in that community issue, correct?

Reply:
Toying around with conda install pinnings, it looks like we actually can do excludes by regex without creating a tight pinning; for example, doing something like:

```
conda install "dask=2022.7.1,!=2022.7.1a.*"
```

Would end up picking up all `2022.7.1.*` packages excluding those that match `2022.7.1a.*`. I assume that we won't be suffixing packages with `a.*` unless they're pre-releases (feel free to point out any cases where we might need this), so happy to try out this option here if it unblocks the security backport proposal 🙂

Reply:
> I assume that we won't be suffixing packages with a.* unless they're pre-releases

Yeah, that's my understanding too